infrastructure/energy-apps-cdk/pyproject.toml

```diff
@@ -4,11 +4,13 @@ version = "0.1.0"
 description = "CDK app for the Energy Apps"
 authors = ["Digital Energy Team via <contentplatform@citizensadvice.org.uk>"]
 readme = "README.md"
+package-mode = false
```

Comment:
A question for my learning: why did `infrastructure/energy-apps-cdk/poetry.lock` and  `infrastructure/energy-apps-cdk/pyproject.toml` also need updating?

Reply:
I ran `poetry run cdk synth` locally to check that the cdk stack built okay, but in order to do that I had to add `package-mode = false` and the `aws-cdk-cli` package to `pyproject.toml`, then run poetry lock to update the `poetry.lock` file.